cmd/mongo.go

```diff
@@ -30,14 +32,17 @@ var (
     # See also 'redis-console' and 'mysql-console'
 `,
 		Action: func(c *cli.Context) {
-			currentApp := detect.CurrentApp(c)
-			opts := db.MongoConsoleOpts{
-				App:  currentApp,
-				Size: c.String("s"),
-			}
 			if len(c.Args()) != 0 {
 				cli.ShowCommandHelp(c, "mongo-console")
-			} else if err := db.MongoConsole(opts); err != nil {
+				return
+			}
+
+			err := db.MongoConsole(db.MongoConsoleOpts{
+				App:          detect.CurrentApp(c),
+				Size:         c.String("s"),
+				VariableName: c.String("e"),
+			})
```

Comment:
Why only MongoDB, IMHO we should support this awesome feature on every databases.